nipype/interfaces/afni/utils.py

```diff
@@ -1696,13 +1697,14 @@ class NwarpCatInputSpec(AFNICommandInputSpec):
     inv_warp = traits.Bool(
         desc='invert the final warp before output', argstr='-iwarp')
     interp = traits.Enum(
+        'wsinc5',
         'linear',
         'quintic',
         'wsinc5',
```

Comment:
what do you mean? this element needs to be removed (so "linear" will be default) or `usedefault=True` should be removed?

Reply:
I have two more traits as this one as `interp` or `ainterp`

Reply:
`'wsinc5'` is listed twice - only once is necessary - for enum's the first element automatically becomes default.

Reply:
oh, you are right, I forgot to remove the second one here. I've changed the order because `Enum` takes the first element as a default, and in the previous version it was `default='wsinc5'`.

Since the previous way of setting default was not working doctests were ignoring these elements, so I'd have to change a few doctests,  unless you believe that these traits shouldn't have default

Reply:
I can create a list of all doctests I'd have to change to pass now the tests

Reply:
sounds good - just don't do `make specs` yet. that should happen after everything has been reviewed.